include/operon/algorithms/ga_base.hpp

```diff
@@ -60,13 +60,23 @@ class GeneticAlgorithmBase {
     [[nodiscard]] auto Elapsed() const -> double { return elapsed_; }
     auto Elapsed() -> double& { return elapsed_; }
 
+    [[nodiscard]] auto IsFitted() const -> bool { return isFitted_; }
+    auto IsFitted() -> bool& { return isFitted_; }
+
     auto Reset() -> void
     {
         generation_ = 0;
         elapsed_ = 0;
         GetGenerator()->Evaluator()->Reset();
     }
 
+    auto RestoreIndividuals(std::vector<Individual> inds) -> void
+    {
```

Comment:
We should probably also check that the provided individuals vector is large enough:

```cpp
 EXPECT(inds.size() == config_.PoolSize + config_.PopulationSize,     
         "Mismatched number of individuals (must match pool/population sizes)");  
```

Of course, `>=` is also possible instead of `==` but this way we make sure that the user knows what they're doing.